src/discordtipbot/discordbot.cr

```diff
@@ -381,18 +382,21 @@ class DiscordBot
       reply(msg, "**ERROR**: There was a problem trying to transfer funds. Please try again later. If the problem persists, please contact the dev for help in #{@config.prefix}support")
     when true
       string = ""
+      amount_each = amount / authors.size
 
       if @tip.get_config(guild_id(msg), "mention")
         authors.each { |x| string = string + ", <@#{x}>" }
       else
         authors.each { |x| string = string + ", #{@cache.resolve_user(x).username}" }
       end
       string = string.lchop(", ")
-      reply(msg, "**#{msg.author.username}** rained a total of **#{amount} #{@config.coinname_short}** onto #{string}")
+      reply(msg, "**#{msg.author.username}** rained a total of **#{amount} #{@config.coinname_short}** (#{amount_each} #{@config.coinname_short} each) onto #{string}")
     end
   end
 
   def active(msg : Discord::Message)
+    return reply(msg, "**ERROR**: Only you are active here!") if private?(msg)
```

Comment:
This does look ok. Not sure if I'd prefer it to be a bit more clear about it being because of being in a DM.

Reply:
This definitely can be improved.

Reply:
In what way?

Reply:
The error is thrown because the channel is a private channel. The error does not tell them this.

Reply:
That's what I was trying to say

Reply:
I guess it's less of an error, and more about returning quickly without actually checking who is active. I could remove the 'error' part and just say 'only you are active'?

Reply:
Just tell them why the error occurred.

"This cannot be done in a private / DM channel." or similar.